### PR TITLE
#1895 add support for external auth

### LINKED ISSUE #1895: Add support for external auth

### PR DESCRIPTION
**Pull request by acbeni**
# Description

Closes #1895

# Checklist

- [x] Update release Notes
- [x] Add to the current github milestone 
- [ ] Update commercetools api reference

### CODE REVIEW

File: commercetools-models/src/main/java/io/sphere/sdk/projects/Project.java [hunk 1]
```diff
@@ -104,6 +104,14 @@
     @Nullable
     ShippingRateInputType getShippingRateInputType();
 
+    /**
+     * If set this property allows to use OAuth2 Bearer tokens issued by another service that provide a RFC 7662-compliant endpoint.
```
Comment (katmatt) [inline commercetools-models/src/main/java/io/sphere/sdk/projects/Project.java]: I would write it as:
```
If this property is set, it allows...
```

File: commercetools-models/src/test/java/io/sphere/sdk/projects/commands/ProjectUpdateActionsIntegrationTest.java [hunk 2]
```diff
@@ -16,8 +20,16 @@
 
 public class ProjectUpdateActionsIntegrationTest extends ProjectIntegrationTest{
 
+    @After
+    public void unsetExternalAuthForPerformanceReasons(){
```
Comment (katmatt) [inline commercetools-models/src/test/java/io/sphere/sdk/projects/commands/ProjectUpdateActionsIntegrationTest.java]: I like this method name 👍

**Comment by acbeni** [comment]
Doc fixed @katmatt

**Comment by alex-ct** [comment]
Looks very good to me 👍

**Pull request merged**